lib/constants/dashboard.js

```diff
@@ -1,3 +1,7 @@
 export const JWT_SECURED = 'JWT_SECURED';
 export const ANY = 'ANY';
 export const OFF = 'OFF';
+
+export const BLANKDASHBOARD = 'blankВashboard';
```

Comment:
There's a typo here which is causing a bug with the blank dashboard icon, and for consistency with the other constants files, I think these names should be amended.

```suggestion
export const BLANK_DASHBOARD = 'blankDashboard';
```

Also, there is another [pull request to rename Curatr to Stream](ttps://github.com/LearningLocker/learninglocker/pull/1476) which when merged with this one, won't update this curatr constant. When that pr is merged, please can this file and the other occurences in this pr be updated to use Stream.